src/modules/Dropdown/Dropdown.story.mdx

```diff
@@ -0,0 +1,139 @@
+
+# Dropdown
+
+This component can be used to select a value from a list. It uses the `Popover` component underneath to show the options.
+
+<Preview>
+  <Story name="Dropdown">
+    <DropdownStory />
+  </Story>
+</Preview>
+
+<Preview>
+  <Story name="Multiple Dropdown">
+    <MultipleDropdownStory />
+  </Story>
+</Preview>
+
+### Available props
+
+All the props are passed through to the `Popover` element, so refer to its documentation for more customization options.
+
+<table width="100%">
+  <thead>
+    <tr>
+      <th>Prop Name</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>items</td>
+      <th>{"IItem: { id: string, text: string }"}</th>
+      <td>
+        The list of options to be presented. This is a generic type, so it may
+        contain additional attributes.
+      </td>
+    </tr>
+    <tr>
+      <td>onItemSelected</td>
+      <th>(item) => void</th>
+      <td>Called when an item is selected.</td>
+    </tr>
+    <tr>
+      <td>selectedItem</td>
+      <th>IItem (optional)</th>
+      <td>The current selection.</td>
+    </tr>
+    <tr>
+      <td>renderHeader</td>
+      <th>() => ReactElement (optional)</th>
+      <td>Use this function to override the header element.</td>
+    </tr>
+    <tr>
+      <td>renderItem</td>
+      <th>(item, selected, index) => ReactElement (optional)</th>
+      <td>Use this function to override the item element.</td>
+    </tr>
+    <tr>
+      <td>header</td>
+      <th>ReactElement (optional)</th>
+      <td>The trigger element. Defaults to a `button`</td>
+    </tr>
+    <tr>
+      <td>onShow</td>
+      <th>Function (optional)</th>
+      <td>Called when the Popover content is shown.</td>
+    </tr>
+    <tr>
+      <td>onShow</td>
+      <th>Function (optional)</th>
+      <td>Called when the Popover content is hidden.</td>
+    </tr>
+    <tr>
+      <td>disabled</td>
+      <th>boolean (optional)</th>
+      <td>If the Popover should not be opened.</td>
+    </tr>
+    <tr>
+      <td>label</td>
+      <th>string (optional)</th>
+      <td>A label to be shown on the header.</td>
+    </tr>
+    <tr>
+      <td>text</td>
+      <th>string (optional)</th>
+      <td>
+        A text to be shown on the header to represent the current selection.
+      </td>
+    </tr>
+    <tr>
+      <td>placeholder</td>
+      <th>string (optional)</th>
+      <td>A text to be shown on the header when no item is selected.</td>
+    </tr>
+  </tbody>
+</table>
+
+#### Multiple Dropdown
+
+If you whish to allow the user to select multiple items use the MultipleDropdown component.
+The props are similar to the Dropdown component except for the following:
+
+<table width="100%">
+  <thead>
+    <tr>
+      <th>Prop Name</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>onItemSelected</td>
+      <th>(items) => void</th>
+      <td>Called when an items are selected.</td>
```

Comment:
`an item is selected` or `when items are selected` depending on whether you want singular or plural (I guess it's the latter)